vet.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -ex  # Exit on error; debugging enabled.
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+die() {
+  echo "$@" >&2
+  exit 1
+}
+
+# TODO: Remove this check and the mangling below once "context" is imported
+# directly.
+if git status --porcelain | read; then
+  die "Uncommitted or untracked files found; commit changes first"
+fi
+# Undo any edits made by this script.
+cleanup() {
+  git reset --hard HEAD
+}
+trap cleanup EXIT
+
+# Only run these tests on Go 1.8 or manually.
+if [[ "$TRAVIS" = "true" && "$TRAVIS_GO_VERSION" != 1.8* ]]; then
```

Comment:
We are going to change this to `1.9*` shortly.
I wonder if there's an easy way to find what the latest go version is...

Reply:
I could find no way.

Since this requires coordination with .travis.yml, how about let's make the script run unconditionally in travis but only launch it conditionally on the version?